### PR TITLE
[ENG-829] updating examples to hooks

### PR DESCRIPTION
**Pull request by yardz**

### CODE REVIEW

**Comment by WarrickFitz** [comment]
I  get the following error when running npm install in the step1 folder. 

image

**Comment by yardz** [comment]
> I get the following error when running npm install in the step1 folder.

I installed with yarn, and in this case there is no error.

This issue is happening because `oreid-login-button` doesn't seem to support react version 18. Actually this dependency seems to only support version 16.
I think we need to open a bug for `oreid-login-button`.

**Comment by WarrickFitz** [comment]
When using yarn I get the following warnings

image

**Comment by WarrickFitz** [comment]
> > I get the following error when running npm install in the step1 folder.
> 
> I installed with yarn, and in this case there is no error.
> 
> This issue is happening because `oreid-login-button` doesn't seem to support react version 18. Actually this dependency seems to only support version 16. I think we need to open a bug for `oreid-login-button`.

Please do open a ticket for this and comment here with the ticket number.

File: examples/react/tutorial/step1-login/README.md [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
-- Retrieving the user's info from the API
-- Using the LoginButton React component
-<br><br>
+## Available Scripts
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app)
+In the project directory, you can run:
 
-    yarn start
+### `npm start`
```
Comment (deckb) [inline examples/react/tutorial/step1-login/README.md:9]: If we are using `yarn` please update the commands here .

File: examples/react/tutorial/step1-login/README.md [hunk 2]
```diff
@@ -1,18 +1,46 @@
+# Getting Started with Create React App
```
Comment (deckb) [inline examples/react/tutorial/step1-login/README.md:1]: Please update the readme to reflect what we are trying to accomplish with this example.

**Review by deckb — CHANGES_REQUESTED** [review]

File: examples/react/tutorial/step1-login/README.md (hunk 1 above)
Comment (yardz) [reply examples/react/tutorial/step1-login/README.md]: Done!

File: examples/react/tutorial/step1-login/README.md (hunk 2 above)
Comment (yardz) [reply examples/react/tutorial/step1-login/README.md]: Done!
I copied the readme that was before!

**Comment by WarrickFitz** [comment]
After cloning into a clean folder and running 

yarn install
yarn start 

in the step1 folder ... it starts, but outputs all these errors

image

**Comment by WarrickFitz** [comment]
Please also add ```yarn install``` to the README

**Comment by yardz** [comment]
> in the step1 folder ... it starts, but outputs all these errors

These errors are from `oreid-js`. I've warned about it a few times, but a tiket was never created.
Any project that adds `oreid-js` will show these errors.

**Comment by yardz** [comment]
> Please also add `yarn install` to the README

Done

**Comment by artmanque** [comment]
@yardz please do a pull and make sure you are using dev/staging environment with the latest changes
you can mark this PR ready when you have completed end-to-end testing

**Review by traylewin — APPROVED** [review]

**Pull request merged**